### ISSUE TITLE
Open Source : 6.26 Explain everything related to `React`

### ISSUE THREAD

**Issue by Aryamanz29** [girlscript/winter-of-contributing#4672]
<hr>

## Description 📜

Open Source : 6.26 Explain everything related to `React`

### NOTE 👇
Please put your doc `6.Popular_Projects` with the name `6.26_React.md`

<hr>

### Domain

Open Source

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by GyaniRoman23** [comment]
Hey @Aryamanz29 I want to work with this issue. Kindly assign it to me.

**Comment by Aryamanz29** [comment]
@GyaniRoman23 Happy learning 🎉🎉

**Comment by GyaniRoman23** [comment]
> @GyaniRoman23 Happy learning 🎉🎉

Thank you:)

**Comment by GyaniRoman23** [comment]
Yeah @Aryamanz29 have been working on it. Its almost finished, will make a PR asap.
